### ISSUE TITLE
pdb file not generated

### ISSUE THREAD

**Issue by lufatatako** [Mixaill/FakePDB#17]
I'm using IDA 7.6, to generate pdb of an executable but it is just creating the json only.
I have tried to create using fakepdb.exe also but that also doesn't works

please help with this issue
thanks

**Comment by Lectem** [comment]
Same issue here, is there some kind of logging I could enable to know what goes wrong ?
The only thing in my output is the following

```
FakePDB/generate pdb:
    * generating JSON: PathToMyDll.dll.json
    * generating PDB: PathToMyDll.pdb
    * symserv EXE id: b''
    * symserv PDB id: b''
    * done
```

**Comment by Lectem** [comment]
I managed to make it work by specifying the exe path myself.
It seems that the tool expects the exe/dll to still be in the same place as it was when the IDA database was created (see 
https://github.com/Mixaill/FakePDB/blob/master/src_plugins/ida/fakepdb/command_pdbgenerate.py#L65), which might not be the case if you share the database with someone (you might even not have the executable file at all).
Is the executable file mandatory to generate a .pdb ?

**Comment by Mixaill** [comment]
Yes, currently it is mandatory, will be fixed in the upcoming release

**Comment by GflowerG** [comment]
+1

ida 7.6 + v0.3 = not generated.
ida 7.4 + v0.2 = ok.

**Comment by sunbeam906** [comment]
I can confirm the same for DLL files. Uploaded the target here: [cshell.zip](https://github.com/Mixaill/FakePDB/files/6956520/cshell.zip).

Using IDA 7.5 SP3 with 0.3. If I try the plugin/tool on an executable for which I've properly adjusted the Debug Directory, it works fine. The tool extracts automatically the RSDS info (GUID, Age) and properly writes the .pdb file to disk, matching it to the .exe information.

If I do it for the DLL I attached above, also with Debug Directory adjusted so it's readable by FakePDB, it fails reading it for some reason. The same situation like lufatatako's above.

Suggestions? Thanks!

**Comment by Mixaill** [comment]
Should be possible with the latest master

[fakepdb_6356b3.zip](https://github.com/Mixaill/FakePDB/files/7042740/fakepdb_6356b3.zip)

**Comment by xNxExOx** [comment]
hello, for me the pdb is also not generated, and no error is printed I tried to message @Mixaill on discord, but no response so far, so I am writing also here

**Comment by sunbeam906** [comment]
> My quick fix based on v0.3 for those who are interested. https://github.com/SoulXHades/FakePDB/releases/tag/v0.3.1

Thank you for taking the time to look into this! Much appreciated!

P.S.: Oh, there are several of us still monitoring this ;)
